packages/apps/docs/src/components/NotFound/NotFound.tsx

```diff
@@ -16,7 +18,7 @@ export const NotFound: FC = () => {
       <Heading as="h5">Not finding the help you need?</Heading>
       <Text>Contact our support desk.</Text>
       <Link href="/help" passHref legacyBehavior>
-        <Button as="a" title="Contact us">
+        <Button as="a" title="Contact us" className={contactLink}>
```

Comment:
I don't think Button accepts a class so I would suspect this doesn't do anything

Reply:
this button is still coming from react-components library stitches! so, it's working fine